core/src/main/scala/cats/syntax/functor.scala

```diff
@@ -1,4 +1,64 @@
 package cats
 package syntax
 
-trait FunctorSyntax extends Functor.ToFunctorOps
+trait FunctorSyntax extends Functor.ToFunctorOps {
+  implicit final def catsSyntaxFunctorTuple2Ops[F[_], A, B](fab: F[(A, B)]): FunctorTuple2Ops[F, A, B] =
+    new FunctorTuple2Ops[F, A, B](fab)
+}
+
+final class FunctorTuple2Ops[F[_], A, B](private val fab: F[(A, B)]) extends AnyVal {
+
+  /**
+   * Lifts `Tuple2#_1` to Functor
+   *
+   * {{{
+   * scala> import cats.Id
+   * scala> import cats.syntax.functor._
+   *
+   * scala> ((1, 2): Id[(Int, Int)])._1F == 1
+   * res0: Boolean = true
+   * }}}
+   */
+  def _1F(implicit F: Functor[F]): F[A] = F.map(fab)(_._1)
+
+  /**
+   * Lifts `Tuple2#_2` to Functor
+   *
+   * {{{
+   * scala> import cats.Id
+   * scala> import cats.syntax.functor._
+   *
+   * scala> ((1, 2): Id[(Int, Int)])._2F == 2
```

Comment:
@johnynek the doc test for `unzip` was ported from the 2.11 branch so I decided to keep to the same type. Tried `Option` but it failed for 2.13 because `Option#unzip` is already defined in 2.13. Switched to `Chain` as it is Cats specific.

Reply:
@johnynek is `Chain` fine or should replace it with something else?